src/components/silhouette-pane.scss

```diff
@@ -0,0 +1,51 @@
+@use './vars';
```

Comment:
Nice -- it looks like we should be replacing our `@import`s with `@use`s across the board.